### PR TITLE
adds lib/dbConnect.js file to connect to Mongo database

### PR DESCRIPTION
**Pull request by Sealcakes**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGODB_URI=''
```
Comment (aarbrown) [inline .env]: Get this file ignored please

Comment (aarbrown) [reply .env]: actually, you may have already committed it

File: app/lib/dbConnect.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    if (cached.conn) {
+        return cached.conn;
+    }
+
+    if (!cached.promise) {
+        const opts = {
+            bufferCommands: false,
+        }
+
+        cached.promise = mongoose.connect(MONGODB_URI, opts).then((mongoose) => {
```
Comment (aarbrown) [inline app/lib/dbConnect.js]: why are we combining promises and async await?

File: package.json [hunk 3]
```diff
@@ -9,8 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "dotenv": "^16.1.4",
```
Comment (aarbrown) [inline package.json]: Is this actually needed?  didn't see it being used.

**Review by aarbrown — CHANGES_REQUESTED** [review]

**Comment by aarbrown** [comment]
@Sealcakes resolve your merge conflict as well

File: app/lib/dbConnect.js (hunk 2 above)
Comment (Sealcakes) [reply app/lib/dbConnect.js]: I was just following the example that Next released on connecting Mongoose
https://github.com/vercel/next.js/blob/canary/examples/with-mongodb-mongoose/lib/dbConnect.js

**Comment by Sealcakes** [comment]
There is going to be a merge conflict with my code and gabe's model code in this file

**Review by aarbrown — APPROVED** [review]

**Pull request merged**